pkg/scheduler/plugins/BUILD

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["registrar.go"],
+    importpath = "k8s.io/kubernetes/pkg/scheduler/plugins",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//pkg/scheduler/internal/cache:go_default_library",
+        "//pkg/scheduler/plugins/v1alpha1:go_default_library",
```

Comment:
How will we plan export this plugins API? should it be part of APIs , e.g. under `scheduler/apis` dir.